packages/web-components/src/components/va-text-input/test/va-text-input.e2e.ts

```diff
@@ -451,6 +451,12 @@ describe('va-text-input', () => {
     expect(el).toHaveClass('usa-input-suffix');
   });
   
+  it('renders an icon if input-icon-suffix is set', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<va-text-input input-icon-suffix=\'credit_card\'></va-text-input>');
+    const vaIconEl = await page.find('va-text-input >>> va-icon');
+    expect(vaIconEl).toHaveClass('hydrated');
```

Comment:
As a suggested improvement, here's how I check that the correct icon is being loaded in the va-button-icon.e2e.ts file:

```
const svgEl = await page.find('va-button-icon >>> va-icon');
const iconProp = await svgEl.getProperty('icon');
expect(iconProp).toMatch('attach_file');

Reply:
@Andrew565  - good call, fixed!